src/main/resources/repos.csv

```diff
@@ -1,8 +1,8 @@
-cloneUrl,branch,org1,org2,org3
-https://github.com/openrewrite/rewrite-recipe-bom,main,,Default,ALL
-https://github.com/openrewrite/rewrite-recommendations,main,,Default,ALL
-https://github.com/Netflix/eureka,master,,Default,ALL
-https://github.com/WebGoat/WebGoat,main,,Default,ALL
+cloneUrl,branch,org1,org2,org3,org4,org5,org6,org7,org8,org9,org10
+https://github.com/openrewrite/rewrite-recipe-bom,main,Default,ALL
+https://github.com/openrewrite/rewrite-recommendations,main,Default,ALL
+https://github.com/Netflix/eureka,master,Default,ALL
+https://github.com/WebGoat/WebGoat,main,Default,ALL

```

Comment:
In order to have GitHub pretty-print this file, the column count has to be the same for all.

```suggestion
cloneUrl,branch,org1,org2,org3
https://github.com/openrewrite/rewrite-recipe-bom,main,Default,ALL
https://github.com/openrewrite/rewrite-recommendations,main,Default,ALL
https://github.com/Netflix/eureka,master,Default,ALL
https://github.com/WebGoat/WebGoat,main,Default,ALL

```

Reply:
I do have to add `org4` as the last entry has 4 orgs listed